community/compatibility-policy.adoc

```diff
@@ -0,0 +1,246 @@
+= Compatibility policy
+Emmanuel Bernard, Yoann Rodiere
+:toc:
+:awestruct-layout: community-standard
+
+Your application interacts with Hibernate libraries,
+and those in turn interact with third-party software: JDBC drivers, other libraries,
+or even REST APIs (Elasticsearch).
+When any of these components changes its API, there is a risk that it becomes incompatible.
+This page explains how Hibernate projects try to limit incompatible changes.
+
+[TIP]
+====
+Hibernate projects comply with this compatibility policy on a best effort basis.
+
+For contractually binding warranty, consider paid support:
++++<br />
+<div class="ui labels blue">
+<a class="ui label" href="/orm/support/"><i class="icon doctor"></i>ORM</a>
+<a class="ui label" href="/search/support/"><i class="icon doctor"></i>Search</a>
+<a class="ui label" href="/validator/support/"><i class="icon doctor"></i>Validator</a>
+<a class="ui label" href="/ogm/support/"><i class="icon doctor"></i>OGM</a>
+</div>
++++
+
+====
+
+== Concepts
+
+[[versioning-scheme]]
+=== Versioning scheme
+
+The first topic when discussing our view on compatibility is how we name versions and the semantics of that.
+
+Hibernate projects are versioned according to the following scheme:
+
+```
+major.minor.micro.qualifier
+```
+
+Where:
+
+* `major`, `minor` and `micro` are incrementing integers.
+* `qualifier` is `Alpha1`, `Alpha2`, etc., `Beta1`, `Beta2`, etc., for development releases,
+`CR1`, `CR2`, etc. for candidate releases
+or `Final` for stable releases.
+
+For example: `4.0.0.Final`, `5.2.0.Beta2`, `5.4.0.CR1`.
+
+For the most part we follow the guidelines as defined by
+https://community.jboss.org/wiki/JBossProjectVersioning[JBoss Project Versioning Guidelines],
+so that is a good background material.
+
+Hibernate, however, applies some additional semantics. Roughly speaking:
+
+* Major updates imply changes to existing APIs.
+* Minor updates simply simply additions to existing APIs.
```

Comment:
```suggestion
* Minor updates simply additions to existing APIs.
```

Reply:
`* Minor updates simply additions to existing APIs.` is not not proper grammar.  And I'm assuming "imply" was meant again:

`* Minor updates imply additions to existing APIs.`

Also, might be good idea to highlight that changes as well as removals fall under "changes".  I think we all understand that, but just to be clear.